.nvmrc

```diff
@@ -1 +1 @@
-v10.18.1
+v14.15.1
```

Comment:
Should this just be the major version, i.e. `v14`, like it is in our other repos?

Reply:
sure